release notes/v0.48.0.md

```diff
@@ -13,7 +13,57 @@ k6 v0.48.0 is here 🎉! This release includes:
 
 ## New features
 
-_optional intro here_
+### Add support for browser module's `page.throttleCPU` [browser#1095](https://github.com/grafana/xk6-browser/pull/1095)
+
+Allows users to throttle the CPU from chrome/chromium's perspective, which helps emulate slower devices when testing the website's frontend. The mandatory argument is of type `CPUProfile`, which is made up of one field named `rate` that is a slow down factor, where `1` means no throttling, and `2` means 2x slowdown, and so on. The documentation for this API can be found [here](https://grafana.com/docs/k6/v0.48.x/javascript-api/k6-experimental/browser/page/throttlecpu).
+
+```js
+...
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    page.throttleCPU({ rate: 4 });
+...
+```
+
+### Add support for browser module's `page.throttleNetwork` [browser#1094](https://github.com/grafana/xk6-browser/pull/1094)
+
+Allows users to throttle the characteristics of the network from chrome/chromium's perspective, which helps emulate poor network connections when testing the website's frontend. The mandatory argument is of type `NetworkProfile` and is defined as:
```

Comment:
I've added more details and a link to the docs in 9d9df6dbb252f7b5aba518349d83fb8fac943366